### ISSUE TITLE
TXS gives "label missing!" error in \ref{label}

### ISSUE THREAD

**Issue by bjtho08** [texstudio-org/texstudio#1176]
### Environment
- TeXstudio: 3.00.00 (git 3.0.0beta2) and 3.00.00 (git 3.0.0beta1)
- Qt: 5.12.1, compiled with Qt 5.12.1 R
- OS: Windows 10
- TeX distribution: texlive

### Expected behavior
A document with the following structure should not give a "label missing!" error:

```latex
\documentclass{article}
\usepackage[utf8]{inputenc}

\title{Test}
\author{Me}
\date{July 2020}

\begin{document}

\maketitle

\section{Test section}

Table \ref{tab:table} is referencing a table in the coming paragraph.

\begin{table}[h]
    \centering
    \begin{tabular}{|l|l|}
        \hline
        A & B \\ \hline
        C & D \\ \hline
    \end{tabular}
    \caption{Table description}\label{tab:table}
\end{table}

\end{document}
```

### Actual behavior
Well, it highlights the label with a wiggly line and the tooltip says "label missing!".

### How to reproduce
Copy/paste the above MWE and verify.

**Comment by muzimuzhi** [comment]
3.0.0beta2 for macOS cannot reproduce your problem. Dose clicking menu item "Idefix -> Refresh Structure" solve the problem on you computer?

**Comment by sunderme** [comment]
works on linux as well.
Can you present a screenshot and the result of help/check active completion files ?

**Comment by bjtho08** [comment]
@muzimuzhi Sorry, "Refresh structure" does not solve the issue on my computer.
@sunderme Sure, I have attached both below.

![txslabel](https://user-images.githubusercontent.com/885853/87820133-cc221e80-c86d-11ea-8095-d0e7fbcd0a7b.png)

[Output from help/check active completion files](https://github.com/texstudio-org/texstudio/files/4939574/completion-output.txt)

**Comment by sunderme** [comment]
fixed

by the way, you should not change the activated completion files manually. txs is doing a good job loading the relevant ones automatically
In your case: 
'biblatex.cwl, biblatex.cwl, class-beamer.cwl, fancyref.cwl, graphicx.cwl, hyperref.cwl, latex-dev.cwl, latex-document.cwl, siunitx.cwl, tex.cwl, tikz.cwl' are activated manually, only latex-dev.cwl, latex-document.cwl, tex.cwl should be globally active. Furthermore there seems to be two biblatex.cwl, probably one user defined/automatically generated. It is probably best only to use the txs delivered one and remove the user/autogenerated version (see wiki where the file is located)

**Comment by bjtho08** [comment]
@sunderme So it was actually an error in class-beamer.cwl? Fancy that. I am pretty sure I activated each one manually years back because they were not properly recognized by txs back then and in the case of biblatex, I think it was missing and I created one manually. Again this is properly about 5 years ago and for every update, including the two betas, I've just installed to the same folder and kept my settings.
But thanks a lot for the help. I appreciate it!
